cuchem/cuchem/benchmark/megamolbart.py

```diff
@@ -36,6 +34,12 @@ def parse_args():
                         type=str,
                         help='Output directory for CSV files')
 
+    parser.add_argument('-u', '--url',
```

Comment:
@rilango Assume this will allow the url to be configurable -- that's good. Can run multiple instances then.